docs/notifications.md

```diff
@@ -65,42 +65,92 @@ Events have a well-defined JSON structure and are sent as the body of
 notification requests. One or more events are sent in a structure called an
 envelope. Each event has a unique id that can be used to uniquely identify incoming
 requests, if required. Along with that, an _action_ is provided with a
-_target, identifying the object mutated during the event.
+_target_, identifying the object mutated during the event.
+
+The fields available in an `event` are described as below.
+
+Field | Type | Description
+----- | ----- | -------------
+ID | string |ID provides a unique identifier for the event.
+Timestamp | Time | Timestamp is the time at which the event occurred.
+Action |  string |  Action indicates what action encompasses the provided event.
+Target | distribution.Descriptor | Target uniquely describes the target of the event.
+Length | int | Length in bytes of content. Same as Size field in Descriptor.
+Repository | string | Repository identifies the named repository.
+FromRepository | string |  FromRepository identifies the named repository which a blob was mounted from if appropriate.
+URL | string | URL provides a direct link to the content.
+Tag | string | Tag identifies a tag name in tag events
+Request | RequestRecord | Request covers the request that generated the event.
+Actor | ActorRecord |  Actor specifies the agent that initiated the event. For most situations, this could be from the authorization context of the request.
+Source | SourceRecord |  Source identifies the registry node that generated the event. Put differently, while the actor "initiates" the event, the source "generates" it.
```

Comment:
I think these field names should be listed by the lowercase (JSON serialied) versions, since this section is documenting the JSON event structure.

It might also be good to define field types like `Descriptor`, `RequestRecord`, `ActorRecord`, and `SourceRecord`, or point to the Go struct for reference. But this is sort of outside the scope of this PR, so it would be fine to come back to improving the documentation in a followup.